configure.ac

```diff
@@ -82,13 +102,25 @@ fi
 if test "$host" = "mingw"; then
   # -static is interpreted by libtool, where it has a different meaning.
   # In libtool-speak, it's -all-static.
-  AX_CHECK_LINK_FLAG([[-static]],[LIBTOOL_APP_LDFLAGS="$LIBTOOL_APP_LDFLAGS -all-static"])
+  AX_CHECK_LINK_FLAG([[-static]],[LIBTOOL_APP_LDFLAGS="$LIBTOOL_APP_LDFLAGS -all-static -no-undefined"])
+fi
+
+if test x$with_net = "xyes"; then
+  AC_CHECK_HEADER([event2/event.h],, AC_MSG_ERROR(libevent headers missing),)
+  AC_CHECK_LIB([event_core],[main],EVENT_LIBS=-levent_core,AC_MSG_ERROR(libevent_core missing))
+  if test "$host" = "mingw"; then
+    AC_CHECK_LIB([event_pthreads],[main],EVENT_PTHREADS_LIBS=-levent_pthreads,AC_MSG_ERROR(libevent_pthreads missing))
+  fi
 fi
 
```

Comment:
Needed to link the math library on x86_64-linux-gnu to build tests
gcc version 8.4.0 (Ubuntu 18.04.6 LTS)

`AC_SEARCH_LIBS([powl],[m],[have_math=yes],[have_math=no])`

Reply:
Will add in the morning. Thanks! 👍